### ISSUE TITLE
Acf builder with sage 10 publish issue

### ISSUE THREAD

**Issue by ashleyvansoeren** [StoutLogic/acf-builder#149]
Hi,
Not sure if this question fits here.
Im trying to use acf builder with sage 10. Installation goes well until i hit wp acorn vendor:publish --provider="Log1x\AcfComposer\Providers\AcfComposerServiceProvider"

the error i get is this:

No publishable resources for tag [].
Publishing complete.

i googled and tried to use the acf.php in config. This doesnt solve it. I also tried wp acorn vendor:publish. This doesnt fix it either. 

I cant use the wp acorn acf:field Example. It wil give me: There are no commands defined in the "acf" namespace. I think this has to do with the publish issue. 

I installed acf in the theme folder. I also configured setup.php to use acf builder. 

I can make the field folder and files manually but i dont get any fields in wordpress.

Does anyone have a solution for me?

Thanks!

**Comment by tomhayes** [comment]
Having the same issue here!

**Comment by Log1x** [comment]
Please open an issue on the ACF Composer repo. ACF Builder is unrelated.

**Comment by mazero** [comment]
Having the same issue @ashleyvansoeren  @tomhayes  @Log1x 
Dou you have find how resolve this to use ACF Builder with sage 10 ? Thx

**Comment by oxfordblucher** [comment]
Having the same issue as well. I've tried Devilbox, WSL, and Local by Flywheel on Windows 10, Windows 11, and Mac OS. I've tried using Acorn as a plugin and as a composer requirement. I've tried different versions of Sage 10, different versions of Acorn, etc. I've tried the solutions in this thread: https://github.com/Log1x/acf-composer/issues/109#issuecomment-1111529061 and while switching the order of 
`$this->app->make('AcfComposer');` & `$this->publishes([])` did allow Log1x\AcfComposer\Providers\AcfComposerServiceProvider to be shown but I am still getting a 'There are no commands defined in the "acf" namespace.' error.
I was finally able to get it working on Devilbox for a couple weeks but it's no longer working after the most recent Mac OS update. I've tried recreating the environment and porting over my preexisting work but the second I add the blocks and fields from the previous project, AcfComposer breaks again. I feel like I'm going insane and I've essentially lost all my work now. Not sure what I'm doing wrong but Sage in general has been a nightmare to work with on Windows.
